src/app/components/story.jsx

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
```

Comment:
The convention for storybook filenames is `[component].stories.jsx`

Also, why is eslint disabled?